src/getCookie.js

```diff
@@ -0,0 +1,9 @@
+exports.getCookie = function getTokenCookie(req) {
+  let cookie = decodeURIComponent(req.headers.cookie);
+  if (cookie === "" || cookie === `access_token=undefined`) {
+    return 0;
+  } else {
+    const token = cookie.split("=");
+    return token[1];
+  }
```

Comment:
Again the abstraction is wrong here. You need to understand data serialization and the data itself are two different things.

NEVER access data from a serialized data source directly.